tests/testthat/test-merge_sce_list.R

```diff
@@ -31,10 +43,10 @@ preserve_rowdata_cols <- "gene_names"
 expected_coldata_cols <- sort(c("sum", "detected", batch_column, cell_id_column))
 
 # Generate some shared data for testing `merge_sce_list()` ------
-sce1 <- sim_sce(n_cells = total_cells / 3, n_genes = total_genes, n_empty = 0)
-sce2 <- sim_sce(n_cells = total_cells / 3, n_genes = total_genes, n_empty = 0)
-sce3 <- sim_sce(n_cells = total_cells / 3, n_genes = total_genes, n_empty = 0)
-sce_list <- purrr::map(
+sce1 <- scpcaTools:::sim_sce(n_cells = total_cells / 3, n_genes = total_genes, n_empty = 0)
+sce2 <- scpcaTools:::sim_sce(n_cells = total_cells / 3, n_genes = total_genes, n_empty = 0)
+sce3 <- scpcaTools:::sim_sce(n_cells = total_cells / 3, n_genes = total_genes, n_empty = 0)
```

Comment:
I think what I meant was to change the library ids of these to be different... `library-1`, `library-2`, etc. Or just `sce1`, `sce2`, etc. which would be more in keeping with how sces are likely to be named.

Then something like the following below for the test.
```
  expect_setequal(
    metadata(result_sce)$library_id,
    c("sce1", "sce2", "sce3")
  ) 
```

Reply:
So I did that... In the `add_sce_data()` at the beginning of the function, I added a `batch` argument. That gets used to create the unique `library_id` and `sample_id`. So when that function is run on the `sce_list`, I pass the name of the sce object to the `batch` option. 

https://github.com/AlexsLemonade/scpcaTools/blob/292aa8a4bb0f3baaf9877e1f8178f058c055b0fb/tests/testthat/test-merge_sce_list.R#L13-L21

Reply:
Ah, I see that now. I was just also expecting an explicit test for the expected `library_ids`, in _addition_ to checking that it matched the `sample_metadata` table. (I was seeing the test in the prepare function which only (correctly) expected a single value).


One more note while we are in there: you could update the following chunks to use `expect_setequal` instead of sorting... https://github.com/AlexsLemonade/scpcaTools/blob/292aa8a4bb0f3baaf9877e1f8178f058c055b0fb/tests/testthat/test-merge_sce_list.R#L178-L193

Reply:
Okay. I added a check for the contents of the `library_id` containing exactly the ids we want and then converted those and any others I saw to `expect_setequal`.